### PR TITLE
Fix pyc anal plugin

### PR DESCRIPTION
**Pull request by FXTi**
**Detailed description**

* Fix jump type error
* Optimize opcode table generation

### CODE REVIEW

**Comment by FXTi** [comment]
This should be merged before https://github.com/radareorg/radare2-extras/issues/244

File: libr/asm/arch/pyc/opcode_27.c [hunk 1]
```diff
@@ -27,10 +27,10 @@ pyc_opcodes *opcode_27 () {
 	name_op (.op_obj = ret->opcodes, .op_name = "IMPORT_NAME", .op_code = 108, .pop = 2, .push = 1); // Index in name list
 	name_op (.op_obj = ret->opcodes, .op_name = "IMPORT_FROM", .op_code = 109, .pop = 0, .push = 1);
 
-	jabs_op (.op_obj = ret->opcodes, .op_name = "JUMP_IF_FALSE_OR_POP", .op_code = 111); // Target byte offset from beginning of code
-	jabs_op (.op_obj = ret->opcodes, .op_name = "JUMP_IF_TRUE_OR_POP", .op_code = 112); // ""
-	jabs_op (.op_obj = ret->opcodes, .op_name = "POP_JUMP_IF_FALSE", .op_code = 114); // ""
-	jabs_op (.op_obj = ret->opcodes, .op_name = "POP_JUMP_IF_TRUE", .op_code = 115); // ""
+	jabs_op (.op_obj = ret->opcodes, .op_name = "JUMP_IF_FALSE_OR_POP", .op_code = 111, .conditional = true); // Target byte offset from beginning of code
+	jabs_op (.op_obj = ret->opcodes, .op_name = "JUMP_IF_TRUE_OR_POP", .op_code = 112, .conditional = true); // ""
+    jabs_op (.op_obj = ret->opcodes, .op_name = "POP_JUMP_IF_FALSE", .op_code = 114, .conditional = true); // ""
```
Comment (wargio) [inline libr/asm/arch/pyc/opcode_27.c:32]: looks like the spacing is wrong here.

**Comment by wargio** [comment]
the build that has failed is r2frida, so is ok for you.

**Comment by FXTi** [comment]
I have formatted all related codes. Code style should be fine now.

**Comment by radare** [comment]
Lgtm!

**Pull request merged**